app/helpers/application_controller_helper.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# rubocop:disable ModuleLength
+module ApplicationHelper
+  include ViewersHelper
+
+  def html_options
+    { class: 'htmlOptions' }
+  end
+
+  # rubocop:disable RescueStandardError
+  def i18n_set?(key)
+    I18n.t key, raise: true
+  rescue
+    false
+  end
+  # rubocop:enable RescueStandardError
+
+def most_focus(data_type, profile_id)
+  data = []
```

Comment:
It would be great to simplify this method and move some of the logic into smaller helper methods